### PR TITLE
About Page Update

### LINKED ISSUE #7: Create an About page
Create an About page that links to the top navigation bar.  Don't worry about content, so much, since I'll probably add something later. Any place holder text will be good for now.

### PR DESCRIPTION
**Pull request by mcdonald-cyber**
Added an About page using a similar template to the tofu page and the search_results.

### CODE REVIEW

File: templates/about.html [hunk 1]
```diff
@@ -1,110 +1,65 @@
...
+
+{% block content %}
+
+<p>Here you can find more wonderful information on the site.<br>
+  What is this site? What does it do? What is it used for?<br>
+  Who built this site? Why did they build this site?<br>
+  How can you help this site continue to exist and grow? (Add donate button?)<br><br>
+
+</p>
+<div>
```
Comment (danieljdorado) [inline templates/about.html]: I think we can remove this `div` and everything in it.

File: templates/about.html [hunk 2]
```diff
@@ -1,110 +1,65 @@
...
-      </form>
-      <div class="row center">
-        {% block content %} {% endblock %}
-      </div>
-      </div>
-      <br><br>
-
     </div>
   </div>
+</div>
```
Comment (apocop) [inline templates/about.html:60]: These `divs` probably should be removed too. I don't see the opening tags.

File: views.py [hunk 3]
```diff
@@ -107,3 +107,8 @@ def tofu(request):
     'View for tofu page.'
     return render(request, 'tofu.html', {'title' : 'Tofu',
                                          'tagline' : 'Not just for eating'})
+
+def about(request):
+    'View function for about page of site.'
```
Comment (danieljdorado) [inline views.py]: The docstring should be in triple quotes.  I'm pretty sure this got copied from my bad example.  😅

File: views.py [hunk 4]
```diff
@@ -107,3 +107,8 @@ def tofu(request):
...
+    return render(request, 'about.html', {'title' : 'About',
+                                          'tagline' : 'Get to know Us'})
```
Comment (danieljdorado) [inline views.py]: Let's make this title case.  Something I also changed in my change list yesterday.   😅  'to' can stay lowercase.

Comment (danieljdorado) [reply views.py]: It look like comment highlights a couple lines so it's not very clear. Opps.  I was talking about the tagline here. :)

File: static/css/materialize.css [hunk 5]
```diff
@@ -1,8 +1,8 @@
-/*!
```
Comment (danieljdorado) [inline static/css/materialize.css:1]: I think these changes were made by accident in this file.  Could you revert them? :)

**Review by danieljdorado — APPROVED** [review]
Thanks for doing this Cynthia!

**Pull request merged**